backend/src/main/java/com/wpz/rbs/service/ActivityService.java

```diff
@@ -94,6 +100,15 @@ public ActivityConflict getConflictsRoom(int roomId) throws ParseException {
     }
 
     public ActivityConflict resolveConflictsRoom(int roomId, List<String> activitiesId) throws ParseException {
+        activitiesId.forEach(activityId -> {
+            Optional<Activity> activity = activityRepository.findById(activityId);
```

Comment:
We can directly look at the reservation tabla - as I remember Id from reservations is passed to activity id.